### PR TITLE
Added support for angular 16

### PR DESCRIPTION
**Pull request by anakin-sw**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"version": 1,
 	"newProjectRoot": "projects",
 	"projects": {
-		"@deliverysolutions/ngx-intl-tel-input-app": {
+		"ngx-intl-tel-input-app": {
```
Comment (rushvora) [inline angular.json:6]: Why change the project name?

File: projects/ngx-intl-tel-input/package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-    "@angular/forms": ">=15.0.0",
-    "google-libphonenumber": "^3.2.3",
-    "intl-tel-input": "^17.0.3",
-    "ngx-bootstrap": ">=10.0.0"
+    "@angular/common": ">=16.2.0",
+    "@angular/core": ">=16.2.0",
+    "@angular/forms": ">=16.2.0",
+    "google-libphonenumber": "^3.2.33",
+    "intl-tel-input": "^18.2.1",
+    "ngx-bootstrap": ">=11.0.2"
```
Comment (rushvora) [inline projects/ngx-intl-tel-input/package.json:10]: The Angular peer dependencies should be greater than `16.0.0`, not `16.2.0`.

**Review by rushvora — CHANGES_REQUESTED** [review]
Reviewed.

File: angular.json (hunk 1 above)
Comment (anakin-sw) [reply angular.json:6]: It's root level app to test package in projects folder. Without this root level testing app does not start

**Pull request merged**